src/mono/wasm/build/WasmApp.InTree.targets

```diff
@@ -1,6 +1,7 @@
 <Project>
   <!-- This depends on the root Directory.Build.targets imported this file -->
   <UsingTask TaskName="MonoAOTCompiler" AssemblyFile="$(MonoAOTCompilerTasksAssemblyPath)" />
+  <UsingTask TaskName="ILStrip" AssemblyFile="$(MonoTargetsTasksAssemblyPath)" />
```

Comment:
Can we give ILStrip a namespace? and use that here? It can be in a follow up PR too.

Reply:
Created an issue to track this https://github.com/dotnet/runtime/issues/89608. iOS app also uses ILStrip. If making the change, would need to update for xamarin ios repo as well. Will take care of that later.